package.json

```diff
@@ -83,7 +83,7 @@
     "monaco-editor": "^0.33.0",
     "ng2-charts": "^4.1.1",
     "ngx-monaco-editor-v2": "^14.0.4",
-    "pdfjs-dist": "3.3.122",
+    "pdfjs-dist": "^4.2.67",
```

Comment:
Can you please check if the `adf-pdf-viewer` component still works as expected? Please check if the pdf is still rendered properly without any erros